utilities/network.py

```diff
@@ -1113,3 +1114,39 @@ def verify_dhcpd_activated(vm):
     except TimeoutExpiredError:
         LOGGER.error(f"{dhcpd} status is not '{active}' but rather '{sample}'")
         raise
+
+
+def verify_nmstate_bridge_creation(bridge: LinuxBridgeNodeNetworkConfigurationPolicy) -> None:
```

Comment:
Why do you need this function anyway?
Creating a bridge should result with a node's bridge marker, so whenever you want to verify bridge creation, you can always call `wait_for_node_marked_by_bridge`.